db/seeds.rb

```diff
@@ -1 +1,100 @@
-load(Rails.root.join("db", "seeds", "#{Rails.env.downcase}.rb"))
+# This seed script populates the development DB with a data set whose size is dependent on the Rails environment.
+
+# You can control the randomness of the data provided by FAKER and the Rails libraries via the DB_SEEDS_RANDOM_SEED environment variable.
+# If you specify a number, that number will be used as the seed, so you can enforce consistent data across runs
+#   with nondefault content.
+# If you specify the string 'random' (e.g. `export DB_SEEDS_RANDOM_SEED=random`), a random seed will be assigned for you.
+# If you don't specify anything, 0 will be used as the seed, ensuring consistent data across hosts and runs.
+
+require "faker"
+require_relative 'seeds/casa_org_populator_presets'
+require_relative 'seeds/db_populator'
+require_relative '../lib/tasks/data_post_processors/case_contact_populator'
+require_relative '../lib/tasks/data_post_processors/contact_type_populator'
+
+class SeederMain
+
+  attr_reader :db_populator, :rng
+
+  def initialize
+    random_seed = get_seed_specification
+    @rng = Random.new(random_seed)  # rng = random number generator
+    @db_populator = DbPopulator.new(rng)
+    Faker::Config.random = rng
+  end
+
+  def seed
+    puts "Erasing all objects from the data base..."
+    destroy_all
+
+    puts "Creating the objects in the data base..."
+    db_populator.create_all_casa_admin('allcasaadmin@example.com')
+    db_populator.create_org(CasaOrgPopulatorPresets.for_environment.merge({org_name: 'Prince George CASA'}))
+    db_populator.create_org(CasaOrgPopulatorPresets.minimal_dataset_options)
+
+    post_process_data
+
+    report_object_counts
+    puts "\nDone.\n\n"
+  end
+
+  private # -------------------------------------------------------------------------------------------------------
+
+  # Used for reporting record counts after completion:
+  def active_record_classes
+    @active_record_classes ||= [
+        CasaOrg,
+        CasaCase,
+        User,
+        Volunteer,
+        Supervisor,
+        CasaAdmin,
+        AllCasaAdmin,
+        SupervisorVolunteer,
+        CaseAssignment,
+        ContactType,
+        ContactTypeGroup,
+        CaseContact,
+    ]
+  end
```

Comment:
picky: 2 spaces for indent

Reply:
Yeah, about those 2 extra spaces... ;)

For many years I have been indenting an extra tab stop to indicate that the line(s) are not a new nesting level, but rather a continuation of the previous line. RubyMine has separate settings for the two; I'm pasting an image of the configuration screen to illustrate (there is an "Indent:" field and a "Continuation Indent:" field). This extra indentation communicates information to the reader that is missing when no such distinction is made. The mandate to erase this distinction, in my opinion, is due to lack of awareness about its benefit, and habit.

That all said, if you all feel strongly about enforcing this code standard, I will conform to it.

![image](https://user-images.githubusercontent.com/28410/96538952-d20e0e00-1267-11eb-8abe-c3021e9aa102.png)

Reply:
I will let the linter enforce whatever it is set for :)